packages/mantine/src/templates/BaseInputTemplate.tsx

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
```

Comment:
Rather than importing from `React`, only import what you need
```suggestion
import { FocusEvent } from 'react';
```